src/atoms/Icon/Icon.tsx

```diff
@@ -1,6 +1,8 @@
 import { size } from 'polished';
 import React from 'react';
-import InlineSVG, { Props } from 'react-inlinesvg';
+import InlineSVG from 'react-inlinesvg';
+// tslint:disable-next-line
+import { Props } from 'react-inlinesvg/lib/types';
```

Comment:
Ran into this issue in `core` and find a solution using `React.ComponentProps<InlineSVG>`